pyvista/core/datasetattributes.py

```diff
@@ -450,21 +401,6 @@ def active_t_coords(self, t_coords: np.ndarray):
         self.SetTCoords(vtkarr)
         self.Modified()
 
-    @property
-    def active_texture_name(self) -> Optional[str]:  # pragma: no cover
-        """Name of the active texture coordinates array.
-
-        .. deprecated:: 0.32.0
-            Use :attr:`DataSetAttributes.active_t_coords_name` to
-            return the name of the active texture coordinates array.
-
-        """
-        warnings.warn(
-            "Use of `active_texture_name` is deprecated. Use `active_t_coords_name` instead.",
-            PyVistaDeprecationWarning,
-        )
-        return self.active_t_coords_name
```

Comment:
I want to point out that is was not supposed to be deprecated... `active_texture_name` is not equivalent to `active_t_coords_name`. `active_texture_name` refers to a PyVista-specific feature I implemented a few years ago that tracks actual `Texture` objects so that if you have an "active texture" that `add_mesh` would pick up on it and render it automatically. This has not equivalent in VTK so I'm fine with it now being broken/not supported but we may want to clean all of this up in a new PR as the `_textures` property this is supposed to control still exists.

cc @pyvista/developers